pkgs/by-name/gz/gz-utils/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  gz-cmake,
+  spdlog,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gz-utils";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gazebosim";
+    repo = "gz-utils";
+    tag = "gz-utils${lib.versions.major finalAttrs.version}_${finalAttrs.version}";
+    hash = "sha256-fYzysdB608jfMb/EbqiGD4hXmPxcaVTUrt9Wx0dBlto=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    gz-cmake
```

Comment:
Should this be in buildInputs or in here?

Reply:
`gz-cmake` is a collection of cmake modules, so I imagine this is correct.